src/prefect/server/schemas/filters.py

```diff
@@ -1567,7 +1567,7 @@ class ArtifactFilterLatest(PrefectFilterBaseModel):
 
     def _get_filter_list(self, db: "PrefectDBInterface") -> List:
         filters = []
-        if self.is_latest is not None:
+        if self.is_latest is not None and self.is_latest:
```

Comment:
Why is the field an `Optional[bool] = None` instead of `bool = False`? `None` has no special meaning for `is_latest` so we should just have it be a boolean with the correct default value.

Reply:
Ah, thanks, I've applied your suggestion 👍